src/black/__init__.py

```diff
@@ -368,6 +377,17 @@ def main(
     config: Optional[str],
 ) -> None:
     """The uncompromising code formatter."""
+    if config and verbose:
+        out(f"Using configuration from {config}.", bold=False, fg="blue")
```

Comment:
I deleted the previous printing of config file location, I think it's better to leave it to --verbose.